### DISCUSSION TITLE
Sparkle using Gigabytes of Space keeping old Messenger Versions

### DISCUSSION THREAD

**Discussion by PaulBetz** [sparkle-project/Sparkle#2495] [Q&A] [answered]
Facebook seems to be using Sparkle to keep Messenger up to date, but the cache folder is now at 7GB on my laptop because Sparkle is keeping all the old Messenger apps. I'm wondering if it is safe to clear out these files to free up space?

**Comment by zorgiepoo** [comment, marked as answer]
Sparkle is supposed to remove a downloaded version after installing it, or when it notices old items in there haven't been modified in 10 days when installing a new update. If there is an issue not resulting them to be removed, it should be safe to remove them. Messenger is also using an old version of Sparkle and logic relating to this has changed.
